Open Judge System/OJS.Services/Business/Problems/ProblemsBusinessService.cs

```diff
@@ -70,32 +66,40 @@ public void RetestById(int id)
 
         public void DeleteById(int id)
```

Comment:
Shouldn't the CRUD-related functionality (Delete in this case) reside in the Data rather than Business service?

Reply:
In this case the process of deletion requires some business logic also, except just deleting the entity.